test/controllers/trails_controller_test.rb

```diff
@@ -51,6 +51,30 @@ class Api::V1::TrailsControllerTest < ActionController::TestCase
                            last["activity_type_name"]
   end
 
+  test "trails have nested species" do
+    trail = Trail.create(name: "howdy hike",
+                         city: "Denver",
+                         state: "CO",
+                         country: "United States",
+                         lat: 10.323,
+                         lng: -103.23,
+                         description: "A darn good time",
+                         directions: "take a right")
+    species = Species.create(kingdom: "Animalia",
+                             trail_id: trail.id,
+                             scientific_name: "Parus caeruleus",
```

Comment:
:+1: 
I'd also like to see this test that the trails and species are associated by lat & long.